CHANGELOG.md

```diff
@@ -85,11 +93,18 @@
 
 ## [v1.0.0](https://github.com/DFE-Digital/dfe-analytics/tree/v1.0.0) (2022-06-10)
 
-[Full Changelog](https://github.com/DFE-Digital/dfe-analytics/compare/v0.1.0...v1.0.0)
+[Full Changelog](https://github.com/DFE-Digital/dfe-analytics/compare/push...v1.0.0)
 
 **Merged pull requests:**
 
 - Bump to v1 [\#15](https://github.com/DFE-Digital/dfe-analytics/pull/15) ([duncanjbrown](https://github.com/duncanjbrown))
+
+## [push](https://github.com/DFE-Digital/dfe-analytics/tree/push) (2022-06-10)
+
+[Full Changelog](https://github.com/DFE-Digital/dfe-analytics/compare/v0.1.0...push)
+
+**Merged pull requests:**
+
```

Comment:
What's happened here? It looks like you still have a `push` tag locally ... can you check and remove if it's there?